packages/opencensus-instrumentation-grpc/src/grpc-stats/client-metrics.ts

```diff
@@ -0,0 +1,70 @@
+import {AggregationType, globalStats, MeasureUnit, View} from '@opencensus/core';
+import {DEFAULT_BYTES_DISTRIBUTION, DEFAULT_MESSAGE_COUNT_DISTRIBUTION, DEFAULT_MILLI_SECONDS_DISTRIBUTION} from './stats-common';
+
+const clientSentMessagesPerRPC = globalStats.createMeasureInt64(
+    'grpc.io/client/sent_messages_per_rpc', MeasureUnit.UNIT,
+    'Number of messages sent in the RPC (always 1 for non-streaming RPCs).');
+const clientSentBytesPerRPC = globalStats.createMeasureInt64(
```

Comment:
We should add a comment ```Measure for total bytes sent across all request messages per RPC.```. Refer -> https://github.com/census-instrumentation/opencensus-java/blob/master/contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcMeasureConstants.java